doc/help/MOUNT.md

```diff
@@ -0,0 +1,41 @@
+kdb-mount(1) - Mount a file to the key database
+===============================================
+
+## DESCRIPTION
+
+This command allows a user to mount a new "backend".  
+A backend allows a file to be interpreted as keys in the key database such that any edits to the keys are reflected in the file and vice versa.  
+Additionally, the user can use this command to list the currently mounted backends by running the command with no arguments.  
+
+## USAGE
+
+`kdb mount [<path> <mountpoint>] [<plugin> [<config>] [..]]`  
```

Comment:
Hello,

Ian Donnelly wrote:
> So is `sw` just short for software?

Yes.

> And if I create keys under
> `user/sw/kdb/` they only affect the current users but `system/sw/kdb/`
> would be all?

Exactly. And `dir/sw/kdb/` would only affect if executed in the same working 
directory.

> How can I find out which settings are available for kdb?

Currently, only by reading the documentation. Later, by looking at 
spec/sw/kdb.

To see changed values, you can use:
`kdb export /sw/kdb simpleini`

best regards,
Markus